src/webgpu/util/f32_interval.ts

```diff
@@ -169,15 +169,15 @@ export type F32Vector =
   | [F32Interval, F32Interval, F32Interval, F32Interval];
 
 /** Coerce F32Interval[] to F32Vector if possible */
-function isF32Vector(v: number[] | IntervalBounds[] | F32Interval[] | F32Vector): v is F32Vector {
+function isF32Vector(v: (number | IntervalBounds | F32Interval)[] | F32Vector): v is F32Vector {
   if (v[0] instanceof F32Interval) {
```

Comment:
This now looks wrong.
`isF32Vector([F32Interval, number, number])` would return true while `isF32Vector([number, F32Interval, number])` would not.

Tests please.

Reply:
Tests and fixed.